packages/core/nuxt-theme-module/index.js

```diff
@@ -35,6 +36,10 @@ module.exports = async function DefaultThemeModule(moduleOptions) {
     });
   }
 
+  this.extendBuild(config => {
```

Comment:
why its moved from nuxt module to theme module? theme module is optional for projects that are not using default theme

Reply:
I remember that something didn't work without that - I'll check if it's still necessary